azure-pipelines.yml

```diff
@@ -97,22 +97,13 @@ jobs:
   - checkout: self
     persistCredentials: true
 
-  - template: azure-pipelines-templates/install-nuget.yml
-
-  - task: DotNetCoreCLI@2
-    displayName: Restore NuGet packages
-    condition: succeeded()
-    inputs:
-      command: restore
-      verbosityRestore: minimal
-      projects: $(System.DefaultWorkingDirectory)/tools/VersionCop/VersionCop.sln
-      feedsToUse: config
-      nugetConfigPath: $(System.DefaultWorkingDirectory)/tools/VersionCop/NuGet.Config
-
-  - script: dotnet build VersionCop.sln -c 'Release' -p:Configuration=Release -p:Platform="Any CPU" -p:PublicRelease=true --no-restore /t:build,pack
+  - script: |
+      dotnet build VersionCop.sln --configuration Release
+      dotnet pack VersionCop.sln --configuration Release
     displayName: Build NuGet package
     condition: succeeded()
     workingDirectory: '$(System.DefaultWorkingDirectory)/tools/VersionCop'
+#  should probably use dotnet publish VersionCop.sln
```

Comment:
Can't do because of authentication on the pipeline. That's why the Azure Pipeline task is used everywhere for publishing packages.